Makefile

```diff
@@ -36,27 +36,6 @@ $(NODE_BIN)/%: package.json package-lock.json
 	npm install
 	touch $(NODE_BIN)/*
 
-$(OAPI_CODEGEN):
```

Comment:
Was this removal intended?  CI is failing due to missing `oapi-codegen` target.

Reply:
Thanks! Yes it is intentional; I miss the pipeline. :+1:

For the idmsvc-api repository we don't need to generate the code, as it is part of the idmsvc-backend.

IMHO linters and openapi validation should be enough on this repository.

I will remove the remaining of this in the pipeline.